library/driver/src/nativeInterop/cinterop/sqlite3mc.def

```diff
@@ -0,0 +1,6 @@
+package = io.toxicity.sqlite.mc.driver.sqlite3mc
+headers = sqlite3mc.h
+
+linkerOpts.linux_x64 = -lpthread -ldl -lm
```

Comment:
NOTE: These are currently expressed `LDFLAGS` for all compilations of `SQLite3MultipleCiphers` via the Makefile (even after configuration). See >> https://github.com/utelle/SQLite3MultipleCiphers/blob/d2be01b9ccff59f20724fc6ea5ba992ca49c96f8/Makefile.am#L29

Will have to play with this in the future as more targets are added, but should only need to add for `macos_x64`.